samples/dxt/Makefile

```diff
@@ -0,0 +1,19 @@
+NXDK_DIR = $(CURDIR)/../..
+SRCS = main.c
+
+LDFLAGS = xbdm.lib
+
+# Hack because nxdk doesn't handle overwriting the entry point.
+# To work around this, we add it as stacksize, which is appended to LDFLAGS.
+NXDK_STACKSIZE=65536 -entry:DxtEntry 
+
+# Hack because bin/ does not exist; this is the first target to avoid an XBE.
+bin/hello.dxt: main.exe
+	@echo "[ BUILD    ] $@"
+	@mkdir -p bin/
+	@cp "$<" "$@"
+
+include $(NXDK_DIR)/Makefile
+
+xbdm.lib: xbdm.dll.def
+	llvm-dlltool -d $< -m i386 -l $@
```

Comment:
We had problems with generating libs with LLVM in the past. This also ignores NONAME, so I'm not very confident this produces a crash-free file.

Reply:
This *is* an issue.

Reply:
Fixed in https://github.com/JayFoxRox/llvm/pull/1